### PR TITLE
[docs] Add notice to avoid confuse

### PR DESCRIPTION
**Pull request by maxchou415**
Latest version of `Nuxt.js` has already included `Vue-Meta` by default.

We don't need to install again, but seems latest version of docs are not mark this message; It makes me really confuse, so I think if we can put this notice in the docs, that can make other developers more friendly.

Thanks.

### CODE REVIEW

**Comment by pimlie** [comment]
Thanks for your contribution!

We already include a message about frameworks which implement vue-meta on the first page of Installation: https://vue-meta.nuxtjs.org/guide/ Also we added a menu item about frameworks with direct links to their docs: https://vue-meta.nuxtjs.org/guide/frameworks.html

Do you feel we still need to add it as well to SSR? If so, shouldnt we also add it to Prepare the plugin?

**Comment by maxchou415** [comment]
Thanks for your replying. 

I think we can add both, for example we can add 
```

```

**Comment by maxchou415** [comment]
Thanks for your replying.

Sorry for my addle-brained, I wasn't read that.

I think we don't need to add in prepare section, but in the SSR section is nice.

Thanks again.

File: docs/guide/ssr.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Server Side Rendering
 
+```
+Latest version of Nuxt.js has already included Vue-Meta by default.
```
Comment (manniL) [inline docs/guide/ssr.md]:
```suggestion
Information: Nuxt.js includes vue-meta by default.
```

**Comment by pimlie** [comment]
Thanks! Unfortunately this is not yet ready for merging, please have a look at the netlify deploy preview. The markup is not nice as it uses a code block for the warning. Use a vuepress `:::tip` for the info block. If in doubt please have a look at Installation. Probably better to copy that message about frameworks and adapt the message to ssr because vue-meta is also used in other frameworks than Nuxt :)

**Pull request closed without merging**